src/clientversion.cpp

```diff
@@ -30,8 +30,10 @@ const std::string CLIENT_NAME("Satoshi");
     #define BUILD_DESC BUILD_GIT_TAG
```

Comment:
Does it make sense to keep `BUILD_GIT_TAG` around? I think it's better to always use the version set in the source rather than whatever the current git tag might be.